x/emissions/keeper/msg_server_update_params.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/zeta-chain/zetacore/x/emissions/types"
+)
+
+func (k msgServer) UpdateParams(goCtx context.Context, msg *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
```

Comment:
It might be a good idea to add some comments, specially around who the expectected authority is